python/federatedml/secureprotol/secretsharing/vss.py

```diff
@@ -0,0 +1,67 @@
+import random
+import gmpy2
+from scipy.interpolate import lagrange
+from federatedml.secureprotol.secretsharing.primes import Primes
+from federatedml.secureprotol import gmpy_math
+
+
+class Vss(object):
+    def __init__(self):
+        self.prime = None
+        self.prime_gen = Primes()
```

Comment:
I found  this is the only ref to class `Primes` which has no usage in other place.
Is Primes useless in this implementation of vss?